src/modules/claims/claims.service.ts

```diff
@@ -1494,12 +1501,35 @@ export class ClaimsService {
     if (!proofVerified) {
       errors.push(ERROR_MESSAGES.PROOF_NOT_VERIFIED);
     }
+    const isExpired = payload?.exp && payload?.exp * 1000 < Date.now();
```

Comment:
I think this is a nice explainer to have here
```suggestion
    // Date.now() and JWT expiration time both identify the time elapsed since January 1, 1970 00:00:00 UTC
    const isExpired = payload?.exp && payload.exp * 1000 < Date.now();
```